package-system/freetype/build_package_image.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root
+# of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+import argparse
+import os
+import platform
+import subprocess
+import sys
+
+parser = argparse.ArgumentParser(description='Builds this package')
+parser.add_argument('--platform', default=platform.system().lower(), required=False, help=f'Platform to build (defaults to \"{platform.system().lower()}\")')
+args = parser.parse_args()
+
+this_dir = os.path.dirname(os.path.realpath(__file__))
+cmake_scripts_path = os.path.abspath(os.path.join(this_dir, '../../Scripts/cmake'))
+
+ly_3rdparty_path = os.getenv('LY_3RDPARTY_PATH')
+ly_android_sdk_path = os.getenv('LY_ANDROID_SDK_PATH');
+
+folder_names = { 
+    #system-name  cmake generation, cmake build
+    'mac'       : ([
+        '-G', 'Xcode',
+         '-DCMAKE_OSX_ARCHITECTURES=x86_64'
+    ], [], 'Debug', 'Release'),
+    'ios'       : ([
+        '-G', 'Xcode',
+        f'-DCMAKE_TOOLCHAIN_FILE={cmake_scripts_path}/Platform/iOS/Toolchain_ios.cmake',
+        '-DPACKAGE_PLATFORM=ios'
+    ], [
+        '--',
+        '-destination generic/platform=iOS'
+    ], 'Debug', 'Release'),
+    'linux'     : ([
+        '-G', 'Ninja Multi-Config',
+        '-DCMAKE_C_COMPILER=clang-6.0', 
+        '-DCMAKE_CXX_COMPILER=clang++-6.0',
+        '-DCMAKE_POSITION_INDEPENDENT_CODE=ON'
+    ], [], 'Debug', 'Release'),
+    'windows'   : ([
+        '-G', 'Visual Studio 16 2019',
+        '-Ax64', '-Thost=x64'
+    ], [], 'Debug', 'Release'),
+    'android'   : ([
```

Comment:
We can get away with only Release configurations on mac/ios/ linux and android since we have no need to debug the freetype library itself.

Reply:
That is correct but I'd rather the script was uniform. Changing it would require retesting every platform and build again.

Reply:
Performance wise, debug builds aren't always efficient for these types of libraries.
This was noticed in the past with debug version of squish-ccr causing processing of specific tiff assets to timeout in debug after an hour on Windows, while the release version version was able to finish asset processing in a timely manner.